lib/elixir/pages/mix-and-otp/config-and-distribution.md

```diff
@@ -0,0 +1,306 @@
+<!--
+  SPDX-License-Identifier: Apache-2.0
+  SPDX-FileCopyrightText: 2021 The Elixir Team
+-->
+
+# Configuration and distribution
```

Comment:
This is great, and helps clarify applications vs supervisors etc. without the complexities of umbrellas.